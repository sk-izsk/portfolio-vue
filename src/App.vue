<template>
  <div id="app">
    <Home />
  </div>
</template>

<script>
import Home from "./screens/Home";
export default {
  name: "App",
  components: {
    Home,
  },
};
</script>

<style></style>
