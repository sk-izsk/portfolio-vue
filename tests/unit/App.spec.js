import { shallowMount } from '@vue/test-utils';
import App from '../../src/App';

describe('App', () => {
  it('renders when passed', () => {
    const wrapper = shallowMount(App);
    expect(wrapper).toMatchSnapshot();
  });
});
