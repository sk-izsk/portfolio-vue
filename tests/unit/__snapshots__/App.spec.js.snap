// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders when passed 1`] = `
<div>
  <!---->
  <div id="app">
    <home-stub></home-stub>
    <navbar-stub></navbar-stub>
    <about-stub></about-stub>
    <education-stub></education-stub>
    <experience-stub></experience-stub>
    <photography-stub></photography-stub>
    <contact-stub></contact-stub>
  </div>
</div>
`;
